src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -12,7 +12,7 @@
     protected void initDefaultCommand() {
         m_robotDrive.driveCartesian(0.0, 0.0, 0.0);
     }
-    public void drive(double x, double y, double theta){
-        m_robotDrive.driveCartesian(x, y, theta);
+    public void drive(DriveCommand command){
+        m_robotDrive.driveCartesian(command.getX(),command.getY(), command.getZ());
```

Comment:
Field absolute drive requires data from the robot pose.
   m_robotDrive.driveCartesian(command.getX(), command.getY(), command.getZ(), pose.getAngle());
Do we need to pass the pose in?  Do we need to filter the pose too?  IE: to turn off field absolute, internally set the pose angle to zero; but if we are filtering the Z to hold the yaw angle, we'll need to true pose angle.  I need to think about the implications of this.